doc/source/getting_started.rst

```diff
@@ -101,13 +103,13 @@ The following outlines how to use the nomenclature package:
 
 **Notes**
 
-* The pyam library is required as *process* takes a *pyam.IamDataFrame* as input.
+* The function :func:`process` takes a :class:`pyam.IamDataFrame` as input.
 
-* *DataStructureDefinition* and *RegionProcessor* are initialized from directories
-  containing yaml files. See :ref:`dir-structure` for details. 
+* :class:`DataStructureDefinition` and :class:`RegionProcessor` are initialized from
+  directories containing yaml files. See :ref:`dir-structure` for details.
 
-* The processor argument of *process* is optional and may only to be used if there are  
-  model mappings. See :ref:`toplevel-functions` for details.
+* The processor argument of func:`process` is optional and may only to be used if there
+  are model mappings. See :ref:`toplevel-functions` for details.
 
 * If not all dimensions of the **DataStructureDefinition** should be validated, a
```

Comment:
```suggestion
* If not all dimensions of the :class:`DataStructureDefinition` should be validated, a
```
In the "Notes" of "Directory structure for definitions and mappings" the classes DataStructureDefinition and RegionProcessor could also be referred as :class:`DataStructureDefinition` and :class:`RegionProcessor` as implemented here if wanted.